src/dfx/src/lib/ledger_types/mod.rs

```diff
@@ -74,16 +73,11 @@ impl fmt::Display for TransferError {
     }
 }
 
-#[derive(CandidType, Deserialize)]
-pub enum CyclesResponse {
-    CanisterCreated(Principal),
-    ToppedUp(()),
-    Refunded(String, Option<BlockHeight>),
-}
-
 /// Position of a block in the chain. The first block has position 0.
 pub type BlockHeight = u64;
 
+pub type BlockIndex = u64;
```

Comment:
What's the difference between a `BlockHeight` and a `BlockIndex` ?

Reply:
I used the exact names from the Candid declaration for clarity. There is no actual difference, even semantically.